content/applications/sales/sales/send_quotations/optional_products.rst

```diff
@@ -1,65 +1,66 @@
-=====================
-Add optional products
-=====================
+=================
+Optional products
+=================
 
-The use of optional products is a marketing strategy for cross-selling products along with a core
-product. The aim is to offer useful and related products to your customers. For instance, if a
-customer wants to buy a car, he has the choice to order an automatic opening trunk and massaging
-seats, or not to order such high-quality products and simply buy his car.
+The use of optional products is a marketing strategy that involves the cross-selling of products
+along with a core product. The aim is to offer useful and related products to customers, which may result
+in an increased sale.
 
-Add optional products to your quotations
-========================================
+For instance, if a customer wants to buy a car, they have the choice to order massaging seats, as
+well, or ignore the offer and simply buy the car. Presenting the choice to purchase optional
+products enhances the customer experience.
+
+Optional products on quotations
+===============================
 
-With Odoo Sales, it is possible to add or modify optional products directly on quotations
-(under the **Optional Products** tab, as you can see below).
+With the Odoo *Sales* application, it is possible to add or modify optional products directly on
+quotations (located under the :guilabel:`Optional Products` tab on a quotation form, in the *Sales*
+application).
 
 .. image:: optional_products/optional_products_2.png
```

Comment:
Note on images - since we are already improving this document, please update the images to follow our guidelines being sure to retaking screenshots if they cannot be resized and still be legible as well as have the correct filename convention (maximum width of 933px or ideally 768px wide & descriptive hyphenated filename). Please let me know if you have any questions! 🙂